src/cdp/cdpClient.ts

```diff
@@ -62,6 +67,10 @@ export class CdpClient extends EventEmitter<CdpEvents> implements ICdpClient {
     this.#sessionId = sessionId;
   }
 
+  get sessionId(): string | undefined {
```

Comment:
sessionId is a public variable of the interface.

Here we are creating a private method in the class, setting it, and returning it. Then we don't use the public one.

Could you please reconcile it in some way? Either use the public one, or remove it.

Reply:
This is not a method it's a getter so that it can't be modified, the interface is correctly implemented this way as well.